concept-amendments.html

```diff
@@ -297,6 +304,23 @@ <h2 id="feeescalation">FeeEscalation</h2>
 <li>It becomes invalid (for example, the <a href="reference-transaction-format.html#lastledgersequence"><code>LastLedgerSequence</code></a> causes it to expire)</li>
 <li>It gets dropped because there are too many transactions in the queue with a higher transaction cost.</li>
 </ul>
+<h2 id="flow">Flow</h2>
+<table>
+<thead>
+<tr>
+<th>Amendment ID</th>
+<th>Status</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>740352F2412A9909880C23A559FCECEDA3BE2126FED62FC7660D628A06927F11</td>
+<td>TBD</td>
+</tr>
+</tbody>
+</table>
+<p>Replaces the payment processing engine with a more robust and efficient rewrite called the Flow engine. The new version of the payment processing engine is intended to follow the same rules as the old one, but occasionally produces different results due to floating point rounding. This Amendment was created to replace the <a href="#flowv2">FlowV2</a> amendment when a critical bug was found in the FlowV2 amendment.</p>
```

Comment:
Propose rewording last sentence:

This Amendment supersedes the <a href="#flowv2">FlowV2</a> amendment.